rules/crowdstrike_rules/event_stream_rules/crowdstrike_password_change.yml

```diff
@@ -0,0 +1,201 @@
+AnalysisType: rule
+Filename: crowdstrike_password_change.py
+RuleID: "Crowdstrike.UserPasswordChange"
+DisplayName: "Crowdstrike User Password Changed"
+Reports:
+  MITRE ATT&CK:
+    - T1098.001 # Additional Cloud Credentials
+CreateAlert: false
```

Comment:
This might be alert-able if UserId != action_target_name

Reply:
Good feedback, I've added that change